hardhat.config.ts

```diff
@@ -60,7 +58,7 @@ task('deploy-gateway', 'Builds and deploys the Gateway contract on the selected
   const { deploy } = await lazyImport('./scripts/deploy-gateway');
   const gatewayDeployment = await deploy(deployments.libs);
 
-  console.log(gatewayDeployment);
+  console.log("Deployed Gateway Actor diamond address: ", gatewayDeployment);
```

Comment:
I thinks this would break `deploy.sh` as we use the output of this task as the input to set the gateway address and automatically deploy in sequence all the IPC contracts. Check that you don't break it before adding this text, or modify `deploy.sh`. That being said, if we end up not using the registry in the end this is not critical.
```suggestion
  console.log(gatewayDeployment);
```

Reply:
Fixed